meta-lmp-base/recipes-core/os-release/os-release.bbappend

```diff
@@ -9,3 +9,9 @@ SUPPORT_URL = "https://support.foundries.io/"
 LMP_MACHINE = "${MACHINE}"
 LMP_FACTORY = "${LMP_DEVICE_FACTORY}"
 LMP_FACTORY_TAG = "${LMP_DEVICE_REGISTER_TAG}"
+
+do_install_append () {
+    if [ -n "${ARCHIVE_DIR}" ] && [ -d "${ARCHIVE_DIR}" ]; then
+        cat os-release > "${ARCHIVE_DIR}"/os-release
```

Comment:
I find a bit weird that we're using ARCHIVE_DIR directly, which is only known to our CI, and I see this same logic as something that can be useful on normal builds.

Would it work if we just deploy at the normal OE deploy folder (${DEPLOYDIR}) and change ci-scripts to move it to archive_dir as done for the normal build artifacts?

Reply:
I guess you mean this https://github.com/foundriesio/ci-scripts/blob/3a09a1c3e248d89cd4c9e9586bfbe30b584c12cb/lmp/build.sh#L50?

I'll try, I agree that it makes more sense.

Reply:
Inside OE the variable that should be used for deployed content is DEPLOYDIR.

Reply:
e.g.

```
do_deploy() {
    install -d ${DEPLOYDIR}
    install -m 0644 boot.scr ${DEPLOYDIR}/boot.scr-${MACHINE}-${PV}-${PR}
    ln -sf boot.scr-${MACHINE}-${PV}-${PR} ${DEPLOYDIR}/boot.scr-${MACHINE}
    ln -sf boot.scr-${MACHINE}-${PV}-${PR} ${DEPLOYDIR}/boot.scr
    install -m 0644 uEnv.txt ${DEPLOYDIR}
}
```